ts/session/apis/snode_api/swarmPolling.ts

```diff
@@ -141,6 +141,12 @@ function mergeMultipleRetrieveResults(
   }));
 }
 
+function swarmLog(msg: string) {
```

Comment:
Nice pattern! I will use this for my stuff too.

Reply:
I kind of want window.log.debug calls to also be controlled by this feature flag?

Reply:
not sure what you meant, but not a todo as part of this PR right?

Reply:
see https://github.com/Bilb/session-desktop/pull/31/files